.github/workflows/checklinks.yml

```diff
@@ -0,0 +1,13 @@
+name: Check Markdown links
+
+on:
+  schedule:
+  - cron: "0 13 * * 1"
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
`main` is the default branch, how release `v3` is recommended to use https://github.com/actions/checkout

Reply:
Thank you so much, I have a question on the v3, when you search for the latest version, it shows  v2.4.2, but when I choose any v3 it tells me to use the latest version instead, so what do you recommend me to do?

## Preview (1)
![image](https://user-images.githubusercontent.com/18606136/191462197-9ba3bdef-b0fb-4bf2-8f93-ecb3aa250e8e.png)

## Preview (2)
![image](https://user-images.githubusercontent.com/18606136/191461754-5d682491-8ca1-4e96-90f3-3c63a1201f72.png)

Reply:
I think any release is better than using a branch